test/runexamplestests.jl

```diff
@@ -26,7 +26,7 @@ instance_sets = [
     ("minimal", Float64, 60),
     # ("minimal", Float32, 60),
     # ("minimal", BigFloat, 60),
-    # ("fast", Float64, 60),
+    # ("fast", Floprat64, 60),
```

Comment:
lmao

Reply:
LOL that should be a technical term for something